packages/jest-cli/src/test_scheduler.js

```diff
@@ -96,6 +96,23 @@ export default class TestScheduler {
         });
         return Promise.resolve();
       }
+      if (testResult.leaks) {
+        const message = [
+          'Your test suite is leaking memory! Please ensure all references are cleaned.',
```

Comment:
I would put a bold statement on top and say "Experimental" etc. Nobody reads `jest --help`.